hawkrest/__init__.py

```diff
@@ -48,14 +48,22 @@ def authenticate(self, request):
         if settings_lookup_name:
             lookup_function = import_string(settings_lookup_name)
 
-        if not request.META.get('HTTP_AUTHORIZATION'):
+        http_authorization = request.META.get('HTTP_AUTHORIZATION')
```

Comment:
if `http_authorization` is None here, meaning that no authorization header was sent at all, this should raise `AuthenticationFailed` regardless of the setting. Otherwise, I see what you mean, this could gracefully defer to other types of `HTTP_AUTHORIZATION` handlers.